chia/plotters/bladebit.py

```diff
@@ -231,7 +290,10 @@ def plot_bladebit(args, chia_root_path, root_path):
             args.connect_to_daemon,
         )
     )
-    plot_type = "ramplot" if args.plot_type == "ramplot" else "diskplot"
+    if args.plot_type == "ramplot" or args.plot_type == "cudaplot":
+        plot_type = args.plot_type
+    else:
+        plot_type = "diskplot"
```

Comment:
would it not be good to have a warning here, if the user specified some unknown plot type?
This will just silently ignore it

Reply:
I'll add the warning